networking/networking.go

```diff
@@ -229,18 +233,34 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
+// GetDefaultIP looks through all loaded networks and returns the Pod's IP
+// from the first network that has IP masquerading enabled.
+// Returns error if no network at all or none with IP masquerading is found.
```

Comment:
I'd just make one sentence: `GetDefaultIP iterates through all loaded networks and returns the IP of the first network having masquerading enabled or an error if none was found.`